src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -234,6 +238,13 @@ class GroupDetails extends React.Component<Props, State> {
   }
 
   renderError() {
+    const {organization, location} = this.props;
+    const projects = organization.projects;
+    const projectId = location.query.project;
+
+    const project = projects.filter(proj => proj.id === projectId);
+    const projectSlug = project[0].slug;
```

Comment:
if we are just looking to get the one project that matches then maybe `find()` is better to use here

```suggestion
    const projectSlug = projects.find(proj => proj.id === projectId)?.slug
```

also unsure if there is a case where projectSlug will be undefined if the user failed to load projects or something weird happens, in that case maybe try and fall back to the original loading message `The issue you were looking for...`

Reply:
awesome! committed that change, and as for the fall back to the other error message, there's a case for a `404`:
<img width="1197" alt="Screen Shot 2020-11-19 at 9 40 07 PM" src="https://user-images.githubusercontent.com/20312973/99763494-d9674800-2aaf-11eb-9c8c-429c9ff4ecf1.png">

and there's also the below error message from the `MissingProjectMembership` component if you try to access a project not part of your organization at all - we can definitely improve this component, but let me know if the other error message is still better for this case

<img width="1137" alt="Screen Shot 2020-11-19 at 9 32 08 PM" src="https://user-images.githubusercontent.com/20312973/99763282-50e8a780-2aaf-11eb-9d69-fb14dfad738d.png">

Reply:
Oh I didn't know about that `MissingProjectMembership` fallback, this looks good then. 

However, this does pose a small typescript problem as the `MissingProjectMembership` component requires a `string` but it is possible that `projects.find(...` returns `undefined`. 

Not sure if others have stronger opinions on this but you may just have to edit the `MissingProjectMembership` code props from `projectId: string` to `projectId?: string` so `undefined` is allowed.

Reply:
Cool, updated this to allow for `undefined`, but let me know if anyone else has any opinion! If not, let me know if this is good to go (for the design, I'll open up another PR for that) :)

Reply:
Yup no worries, the `undefined` thing is probably fine as the component catches it anyway so this PR should be good